solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -1045,4 +1114,7 @@ contract InterchainAccountRouter is Router {
                 address(hook)
             );
     }
+
+    /// @dev This allows us to receive gas refunds from the IGP
+    receive() external payable {}
```

Comment:
I guess without this, refunds would revert which means that overpayment would revert
that seems preferable